cogs/auto_standup.py

```diff
@@ -0,0 +1,45 @@
+import discord
+import asyncio
+import datetime
+from datetime import timezone
+from discord import app_commands
+from discord.ext import commands
+
+channel_ID = 1172727647382552737
```

Comment:
Just wondering, is this channel_ID hardcoded to the bot-testing-spam in WIT Willowbot?